### PR TITLE
Update train.py

### PR DESCRIPTION
**Pull request by msehabibur**
Currently, the train data is not correctly printed out in the prediction_results_train_set.csv file. It shows some symmetric data points along the parity line. Also, it prints out the validation dataset rather than the training dataset. Target and prediction values are alternated every other row, and the training set predictions are replicated.

### CODE REVIEW

**Comment by bdecost** [comment]
@knc6 this PR address #88

- the file `prediction_results_train_set.csv` will now contain training targets and predictions (instead of validation set)
- the duplicate flipped line (missed debugging output?) is removed

One potential addition: serialize validation set targets/predictions?

**Comment by msehabibur** [comment]
Thank you so much @bdecost for your great effort

**Review by bdecost — APPROVED** [review]
this now reads training data from the trainEOS epoch store, and removes the apparent debug output from the CSV serialization
